src/lib/crypto/mpi.cpp

```diff
@@ -43,7 +43,11 @@ to_buf(buf_t *b, const uint8_t *in, size_t len)
 const buf_t
 mpi2buf(pgp_mpi_t *val, bool uselen)
 {
-    return (buf_t){.pbuf = val->mpi, .len = uselen ? val->len : sizeof(val->mpi)};
+    return
```

Comment:
Hm, don't know how these survived code changes, but type buf_t and mpi2buf and all other functions are not used anywhere so could be safely removed.